src/crypto.ts

```diff
@@ -1,5 +1,4 @@
 import { misc, codec } from "sjcl";
-import { Buffer } from "buffer";
```

Comment:
As weird as it looks, having this line in this file, though unused, was preventing a bug in the browser. This should be tested before we merge it in - I can do that by running the test-skapp tests when I have time.

Reply:
With latest commits, I think this is fixed.